src/dsp_tools/cli/entry_point.py

```diff
@@ -90,13 +96,24 @@ def _check_version() -> None:
         return
     latest = parse(response.json()["info"]["version"])
     installed = parse(version("dsp-tools"))
-    if latest > installed:
-        msg = colored(
-            f"You are using DSP-TOOLS version {installed}, but version {latest} is available. "
-            "Consider upgrading via 'pip3 install --upgrade dsp-tools'.",
-            color="red",
-        )
-        print(msg)
+    if latest == installed:
+        return
+
+    msg = (
+        f"You are using DSP-TOOLS version {installed}, but version {latest} is available. "
+        "Consider upgrading via 'pip3 install --upgrade dsp-tools'. "
+    )
+    if latest.base_version == installed.base_version:
+        print(colored(msg, color="red"))
+        return
+
+    msg += "Continue anyway? [y/n]"
```

Comment:
can we put this on a new line? I tried it out and it can easily be overlooked.

Reply:
Done